web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/ArgsUriPart.java

```diff
@@ -202,20 +159,6 @@ public void reset(Class declaringInterface, String resource, MethodTransfilter m
 		slurp = true;
```

Comment:
if the possibility to change it via setSlurpQueryParameter() is dopped, there would also be no need to reset it.